shared/request_caching.go

```diff
@@ -75,8 +77,13 @@ type cachingHandler struct {
 	cache       ReadWritable
 	isCacheable func(*http.Request) bool
 	getCacheKey func(*http.Request) interface{}
+	shouldCache func(int) bool
 }
 
+func defaultIsCacheable(*http.Request) bool          { return true }
+func defaultGetCacheKey(r *http.Request) interface{} { return r.URL.String() }
```

Comment:
And this `URLAsCacheKey`:
https://github.com/web-platform-tests/wpt.fyi/pull/674/files#diff-6c9911785ad4f162692fbb1bcc25984aR158

Reply:
Patched that in and added `CacheStatusOK` for the third default function.